File: src/main/java/ro/ubb/andreeaLung/src/lab1/Factory/Cappuccino.java
package lab1.Factory;

import lab1.SOLID.Ingredient;

import java.util.List;

public class Cappuccino implements Coffee {
    List<Ingredient> ingredients;

    public Cappuccino() {
        this.ingredients = Coffee.cappuccino;
    }

    @Override
    public CoffeeType getType() {
        return CoffeeType.CAPPUCCINO;
    }

    @Override
    public List<Ingredient> getIngredients() {
        return this.ingredients;
    }
}

File: Laboratoare_gbir2109/DP_Lab1_Homework/DP_Lab1_Homework/Models/Cappuccino.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DP_Lab1_Homework.Models
{
    public class Cappuccino : BaseCoffee
    {
        public Cappuccino()
        {
            Ingredients = new List<Ingredient>();
            CoffeeType = Coffee.Cappuccino;
        }
    }
}
